src/config/docs.py

```diff
@@ -18,3 +20,8 @@ class DocsConfig(BaseModel):
 
     GENERATE_UNTIL_FORK: str = "Osaka"
     """The fork until which documentation should be generated."""
+
+    DOCS_BASE_URL: str = f"https://ethereum.github.io/execution-spec-tests/v{AppConfig().version}"
+
+    # Documentation URLs prefixed with `DOCS_URL__` to avoid conflicts with other URLs
+    DOCS_URL__WRITING_TESTS: str = f"{DOCS_BASE_URL}/writing_tests/"
```

Comment:
Great idea to use the config for this.

I think I'd prefer to point to `main` (for now) as we're pushing improvements to the docs much faster than we are releasing.